frontend/src/components/experimentRuns/ExperimentRunValidation.tsx

```diff
@@ -146,6 +151,23 @@ interface LanePanelProps {
 }
 
 function LanePanel({ lane, canValidate, canReset, isValidationInProgress, setPassed, setFailed, setAvailable }: LanePanelProps) {
+	const dispatch = useAppDispatch()
+  const datasetsById = useAppSelector(selectDatasetsByID)
+	const [datasets, setDatasets] = useState<Dataset[]>([])
+
+  useEffect(() => {
+    Promise.all(lane.datasets.map(async (dataset) => {
+      const response = await dispatch(api.datasets.get(dataset.datasetID))
+      return response.data
+    }))
+    .then((values) => {
+      setDatasets([...values])})
```

Comment:
why deconstruct an array when it's a direct substitution? i believe it could be done simply with setDatasets(values)